docs/schemas/core/IfcKernel/Entities/IfcObject.md

```diff
@@ -58,6 +58,7 @@ Set of relationships to property set definitions attached to this object. Those
 ## Formal Propositions
 
 ### UniquePropertySetNames
+If the object has property sets, their names must be unique.
```

Comment:
```suggestion
Every individual _IfcPropertySetDefinition_ assigned to the object using _IfcRelDefinesByProperties_ shall have a unique _Name_ attribute value.
```

Leaning on `IfcComplexPropertyTemplate.UniquePropertyNames`, but adapted to the context:

1. I believe that more explicit documentation would make it easier for the reader. 
2. This makes it more consistent with very verbose documentation in #644, for example.
3. "object *has* property sets" is not the best choice of words IMO. [Documentation](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcRelDefinesByProperties.htm) uses *assignment*; `IsDefinedBy` uses *attached*.